.github/renovate.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>podium-lib/renovate-presets:top-level-module"],
```

Comment:
Use https://github.com/finn-no/renovate-presets instead of podium-lib. Sorry if I put you wrong on that one!